js/main.js

```diff
@@ -28,26 +29,31 @@ function getCSVData(callback = undefined) {
   });
 }
 
+function getCardsByRarity() {
```

Comment:
(nit) add a comment explaining the shape of the returned result, ie
```
{ "Common": [ {card_name: "snailtare", rarity: "common",...}, {card}, {card}...]
  "Uncommon": [{card}],
   ....
}
```

Reply:
Done o7